.github/workflows/sync-build-cache.yml

```diff
@@ -0,0 +1,102 @@
+name: Sync build cache
+
+permissions:
+    actions: write
+    contents: read
+
+on:
+    schedule:
+        - cron: '0 22 * * *'
+    workflow_dispatch:
+
+env:
+    NODE_OPTIONS: '--max-old-space-size=8192'
```

Comment:
Same as here, is this space sufficient?

cc: @habib-deriv @balakrishna-deriv

Reply:
Yes should be fine, locally it takes less than this @adrienne-deriv